src/plus/webviews/graph/graphWebview.ts

```diff
@@ -140,9 +141,13 @@ export class GraphWebview extends WebviewWithConfigBase<State> {
 
 		let commits: GitCommit[] | undefined;
 		if (ref != null) {
-			const commit = await this.selectedRepository?.getCommit(ref);
-			if (commit != null) {
-				commits = [commit];
+			if (ref === 'work-dir-changes') {
+				// TODO: Add action for selecting WIP row.
+			} else {
```

Comment:
@eamodio @d13 I left the WIP row selection logic as TODO because I'm not sure yet how to interact with the SCM view and for the sake of speed, I was wondering if either of you could pick this part up when you have a chance. 🙏🏼